dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md

```diff
@@ -0,0 +1,625 @@
+
+## Item Attributes
+Item attributes in the request allow you to include item-specific values like a shipping group or an origin. They are required if you are using any type of features such as carrier rules, dimensional shipping, multi-origin, etc. The most common attributes are listed below. 
+
+
+:::tip Values are case sensitive and require delimiter
+All values are case sensitive and require a delimiter between multiple values. The default delimiter is a comma, but if your data is already separated by hashes (#), it is required for you to set the appVersion to 2.0.0 within the siteDetails.
+
+:::
+Attribute Name	Description
+shipperhq_shipping_group	This is used to place the shipping group on an item to match carrier rules.
+shipperhq_warehouse	This is used if you are using multi-origin functionality in order to specify which origin(s) are applicable for an item. These values are separated by the “#’ character and must match origin names configured in ShipperHQ.
+shipperhq_dim_group	This is used if you have created any dimensional rules for your items to be packed in a specific way.
+ship_length, ship_width, ship_height	These are used if you are placing dimensions on items; Note, you must include all 3 attributes when using this query.
+ship_separately	This allows for the item to be packed separately into its own box when used for rating.
+```json title="Attribute example"
+    "attributes" : [ {
+        "name" : "shipperhq_shipping_group",
+        "value" : "FLAT59,FREE"
+      }, {
+        "name" : "ship_width",
+        "value" : "2.0000"
+      }, {
+        "name" : "ship_length",
+        "value" : "2.0000"
+      }, {
+        "name" : "ship_height",
+        "value" : "36.0000"
+      }, {
+        "name" : "shipperhq_dim_group",
+        "value" : "RULE1,RULE2"
+      }, {
+        "name" : "shipperhq_hs_code",
+        "value" : "123456"
+      }, {
+        "name" : "shipperhq_warehouse",
+        "value" : "ORIGIN1,ORIGIN2"
+      } ], 
+```
+
+## Requested Options
+
+Requested Options in the requestedOptions field in the request allow you to include services for the entire shipment such as Liftgate or Residential delivery. These are useful if you are using Freight carriers or small package carriers that offer specific delivery methods for residential delivery. The most common attributes are listed below. 
+
+::: tip All values are case-sensitive
+:::
+
+Requested Options	Data Type	Description
+liftgate_required	Boolean	This specifies liftgate is required at the destination.
+notify_required	Boolean	This requests an appointment delivery or notice of delivery.
+inside_delivery	Boolean	This requests delivery inside the destination.
+limited_delivery	Boolean	This specifies limited access at the destination address.
+destination_type	Boolean	This specifies if a destination is a Residential or Business address and is used for freight and small package shipments.
```

Comment:
We probably want to re-format this. Maybe add a table and backticks for code?

Reply:
Yep